CesiumGeospatial/test/TestS2CellID.cpp

```diff
@@ -92,7 +92,7 @@ TEST_CASE("S2CellID") {
     center = S2CellID::fromToken("5").getCenter();
     CHECK(Math::equalsEpsilon(
         center.longitude,
-        Math::degreesToRadians(-180.0),
+        Math::degreesToRadians(0),
```

Comment:
While the other change from -180 to 180 is innocuous (they both represent the same location on a globe) the change here from -180 to 0 implies the semantic meaning of specific tokens has changed in s2.  I'm not sure if cell ids are used between different Cesium network components, but if they are this would imply a problem.